spec

```diff
@@ -2155,7 +2155,6 @@ endseg
 beginseg
     name "ovl_En_Daiku"
     include "build/src/overlays/actors/ovl_En_Daiku/z_en_daiku.o"
-    include "build/data/overlays/actors/z_en_daiku.data.o"
     include "build/data/overlays/actors/z_en_daiku.reloc.o"
```

Comment:
Auto generated reloc should be used in `NON_MATCHING` (you can check references to `NON_MATCHING` in the spec if you're not sure how to do it)
```suggestion
    include "build/data/overlays/actors/z_en_daiku.reloc.o"
```